modules/react/breadcrumbs/lib/hooks/useBreadcrumbsModel.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {createModelHook} from '@workday/canvas-kit-react/common';
+import {defaultGetId, useOverflowListModel} from '@workday/canvas-kit-react/collection';
+import {useMenuModel} from '@workday/canvas-kit-react/menu';
+import {reorganizeHiddenItems} from './utils';
+
+export const useBreadcrumbsModel = createModelHook({
+  defaultConfig: {
+    ...useOverflowListModel.defaultConfig,
+    /**
+     * Optional id for the whole `ActionBar` group. If not provided, a unique id will be created.
+     * @default useUniqueId()
+     */
+    id: '',
+    /**
+     * The default ActionBar sub-components only handle rendering of button group in a horizontal orientation,
+     * but the sub-components could be replaced to handle vertical orientations.
```

Comment:
Can we have an vertical breadcrumbs? what does that look like?

Reply:
No, we don't. We have only horizontal breadcrumbs, but we still keep opportunity to change direction for this hook if it would be needed.